specs/eventing/broker.md

```diff
@@ -145,6 +145,18 @@ The subscriber MAY receive a confirmation that a reply event was accepted by the
 Broker. If the reply event was not accepted, the initial event SHOULD be
 redelivered to the subscriber.
 
+Subscribers SHOULD acknowledge delivery of an event with an HTTP 2xx response code.
+
+Subscribers MAY respond to event delivery with one of the following responses
```

Comment:
Upon successful delivery, subscribers MUST respond with one….

Reply:
Here is where I think Scott really took issue - that the specific codes are recommendations and not requirements.

Reply:
Some variable of "MUST respond with one of...." doesn't mandate which 2xx, just makes it clear it must be a 2xx.